### PR TITLE
Add the need for a final communication before concluding the report

### PR DESCRIPTION
**Pull request by malvikasharan**
The current steps do not explicitly talk about why a final meeting with the reportee is important for both the IRG and reportee.

Added a sentence to clarify that.

### CODE REVIEW

**Comment by malvikasharan** [comment]
Tagging @fmichonneau and @karenword

**Comment by maneesha** [comment]
Thanks @malvikasharan 
Seeking approval from @carpentries/executive-council before merging

File: topic_folders/policies/enforcement-guidelines.md [hunk 1]
```diff
@@ -73,6 +74,7 @@ When following up with the reportee, the CoCc representatives will:
 - Give them the opportunity to state their view of the incident.
 - Remind them of the consequences of their behaviour, or future consequences if the behaviour is repeated.
 - Explain the possible resolutions that may be enforced should the CoCc determine there is a breach.
+- A final meeting/email-communication will ensure that reportee understands the resolution and has an opportunity to clarify concerns.
```
Comment (fmichonneau) [inline topic_folders/policies/enforcement-guidelines.md:77]: Start the sentence with a verb and clarify that email communication is a secondary means of communication:
```suggestion
- Conduct a final meeting to ensure that the reportee understands the resolution and has an opportunity to clarify concerns. In some cases, this final communication could be carried over email at the discretion of the IRG.
```

Comment (malvikasharan) [reply topic_folders/policies/enforcement-guidelines.md:77]: Looks good. Thank you for the edit.

**Comment by lexnederbragt** [comment]
Delegating this to @konrad who may approve it on behalf of the EC.

**Comment by konrad** [comment]
As I am transitioning out of the CoC as liaison and @masamiy took over recently I will discuss this with her and we solve this together.

**Comment by masamiy** [comment]
> As I am transitioning out of the CoC as liaison and @masamiy took over recently I will discuss this with her and we solve this together.

Thanks @konrad, it looks good to me, too!

**Comment by konrad** [comment]
Thanks, @malvikasharan. The suggestion is approved by @masamiy and me.

**Comment by maneesha** [comment]
Thanks all.   @malvikasharan  is this ready to merge?

**Comment by maneesha** [comment]
@malvikasharan is this ready to merge?

**Comment by maneesha** [comment]
Merging this as @konrad has approved after being delegated by @lexnederbragt 

Feel free to re-open if there are any concerns.

**Pull request merged**